Kconfig.zephyr

```
@@ -292,6 +292,17 @@ config LINKER_LAST_SECTION_ID_PATTERN
 	  be used.
 	  The size of the pattern must not exceed 4 bytes.
 
+config LINKER_USE_RELAX
+	bool "Use --relax Linker option"
+	default y
```

with the corresponding `NO_RELAX`, then a depends must be added here:
```suggestion
	depends on !LINKER_USE_NO_RELAX
	default y
```